iceoryx_utils/test/moduletests/test_message_queue.cpp

```diff
@@ -239,7 +243,11 @@ TEST_F(MessageQueue_test, sendAfterServerDestroy)
     std::string message = "Try to send me";
     auto sendResult = client.send(message);
     EXPECT_TRUE(sendResult.has_error());
+#if defined(__APPLE__)
+    EXPECT_THAT(sendResult.get_error(), Eq(IpcChannelError::CONNECTION_RESET_BY_PEER));
```

Comment:
it's unfortunate to have different error codes depending on the channel type. This make error handling more error prone if you exchange the IpcChannelType. Is there a possibility to find a common error code for this behaviour?

Reply:
The `IpcChannelError` value depends on the implementation but not that we expect some kind of error. I would suggest to remove our expectation for a specific kind of error and just expect one.